hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -1309,7 +1309,7 @@
   <property>
     <description>Environment variables that containers may override rather than use NodeManager's default.</description>
     <name>yarn.nodemanager.env-whitelist</name>
-    <value>JAVA_HOME,HADOOP_COMMON_HOME,HADOOP_HDFS_HOME,HADOOP_CONF_DIR,CLASSPATH_PREPEND_DISTCACHE,HADOOP_YARN_HOME,HADOOP_HOME,PATH,LANG,TZ</value>
+    <value>JAVA_HOME,HADOOP_COMMON_HOME,HADOOP_HDFS_HOME,HADOOP_CONF_DIR,CLASSPATH_PREPEND_DISTCACHE,HADOOP_YARN_HOME,HADOOP_HOME,PATH,LANG,TZ,LD_LIBRARY_PATH</value>
```

Comment:
I'm -1 on adding LD_LIBRARY_PATH in the white list by default since [it is considered to be avoided usually](https://www.hpc.dtu.dk/?page_id=1180#:~:text=Inconsistency%3A%20This%20is%20the%20most,compatible%20with%20the%20original%20version.). Users should opt-in by changing the `yarn.nodemanager.env-whitelist`.